project/src/pages/main/main-screen.tsx

```diff
@@ -72,7 +73,7 @@ export default function MainScreen({cardsCount}: MainScreenProps): JSX.Element {
               </form>
 
               <div className="cities__places-list places__list tabs__content">
-                { Array.from([...Array(cardsCount)], (item) => <Card key={item} />) }
+                { Array.from([...Array(cardsCount)], (_item, index) => <Card key={index} />) }
```

Comment:
Можно лучше, обрати внимание на форматирование
```suggestion
                {Array.from({ length: cardsCount }, (_, index) => <Card key={index} />))}
```